ddtrace/appsec/_common_module_patches.py

```diff
@@ -47,7 +47,14 @@ def wrapped_open_CFDDB7ABBA9081B6(original_open_callable, instance, args, kwargs
 
         check_and_report_path_traversal(*args, **kwargs)
 
-    if asm_config._asm_enabled and asm_config._ep_enabled:
+    from ddtrace import tracer
```

Comment:
Is it possible to move these imports to top-level?

Reply:
done.